scripts/generate_conda_file.py

```diff
@@ -55,14 +55,15 @@
     "swig": "swig==3.0.12",
     "tensorflow": "tensorflow==1.12.0",
     "lightgbm": "lightgbm==2.2.1",
+    "cmake": "cmake==3.14.0",
```

Comment:
is it typical for CONDA to have the libraries in pseudorandom order compared to alphabetical?

Reply:
yeah, here we just put a random order, no specific reason, but probably an alphabetical order would be nicer